### PR TITLE
Use maas ui table components

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Use maas ui table components

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Sanity check all the things

## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/844

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,11 +1,5 @@
 {
-  "extends": [
-    "react-app",
-    "airbnb",
-    "prettier",
-    "prettier/react",
-    "plugin:jest/all"
-  ],
+  "extends": ["react-app", "prettier", "prettier/react", "plugin:jest/all"],
```
Comment (barrymcgee) [inline .eslintrc]: @hatched Bundled the removal of Airbnb config here otherwise there was 🚨everywhere.

File: src/components/TableSet/TableSet.js [hunk 2]
```diff
@@ -90,32 +94,32 @@ const tableSet = {
...
-    return <tr key={row.model}>{returnTableBodyData(row)}</tr>;
+    return <TableRow key={row.model}>{returnTableBodyData(row)}</TableRow>;
   });
 };
 
 const renderTables = tablesObj => {
   const tables = Object.keys(tablesObj);
   return tables.map((key, i) => (
-    <table key={tables[i]}>
+    <Table key={tables[i]}>
```
Comment (anthonydillon) [inline src/components/TableSet/TableSet.js]: Maybe try out the sortable?

Comment (barrymcgee) [reply src/components/TableSet/TableSet.js]: Just added it.

**Comment by barrymcgee** [comment]
Note: just realised maas-ui uses a container `MainTable` component to generate the tables and add the interactivity. 

We use `TableSet` to generate the tables but do not yet add the sortable etc.

I'll add that as a follow-up PR.

**Comment by hatched** [comment]
@barrymcgee can you add `.DS_Store` to the `.gitignore` plz as there is some in this PR.

**Comment by hatched** [comment]
I see that there is a sortable attr on the table but there don't appear to be any UI elements or functionality to go with that.

**Comment by barrymcgee** [comment]
@hatched It adds the classname and styling needed for the next part - the functionality.

**Pull request merged**